docs/inputs/index.md

```diff
@@ -9,15 +9,17 @@ provided, a dropdown field will appear on the Inputs page. In contrast, a button
-| [table](../table.md)                                                      | object | It displays input stanzas in a tabular format.                                                        |
-| groupsMenu                                                                | array  | This property allows you to enable the [multi-level menu](./multilevel_menu.md) feature.              |
-| [services](#services-properties)<span class="required-asterisk">\*</span> | array  | It specifies a list of modular inputs.                                                                |
+| Property                                                                  | Type   | Description                                                                                                                              |
+|---------------------------------------------------------------------------|--------|------------------------------------------------------------------------------------------------------------------------------------------|
+| title<span class="required-asterisk">\*</span>                            | string | -                                                                                                                                        |
+| description                                                               | string | It provides a brief summary of an inputs page.                                                                                           |
+| [subDescription](../advanced/sub_description.md)                          | object | It provides broader description of an inputs page.                                                                                       |
+| menu                                                                      | object | This property allows you to enable the [custom menu](../custom_ui_extensions/custom_menu.md) feature.                                    |
+| [table](../table.md)                                                      | object | It displays input stanzas in a tabular format.                                                                                           |
+| groupsMenu                                                                | array  | This property allows you to enable the [multi-level menu](./multilevel_menu.md) feature.                                                 |
+| [services](#services-properties)<span class="required-asterisk">\*</span> | array  | It specifies a list of modular inputs.                                                                                                   |
+| readonlyFieldName                                                         | string | Name of the boolean attribute that UCC checks for each input. If the attribute is true for an input, editing is disabled for that input. |
```

Comment:
I initially thought about hardcoding, but that would create conflicts if some some addon actually uses this field for other purpose.

Reply:
another option is having a boolean field in globalConfig to check `hard_disabled` or not. This would eliminate the risk for addons that don't use this feature, but it would be the same effort, but less flexibility.

Reply:
> so maybe worth removing that part about boolean attribute and add truthy attribute.

then we have to define truthy attributes..

<img width="221" alt="image" src="https://github.com/splunk/addonfactory-ucc-generator/assets/142901247/a1f7fe0b-b807-4a90-b9ac-12be77e5181b">

Reply:
true, but also we can just add link in docs for ie. this  https://developer.mozilla.org/en-US/docs/Glossary/Truthy and be more trustful that they will use it right :D. But yea maybe simple boolean seems like the best option for now